Update muon interface docs (#25781)
mantidproject/mantid: docs/source/interfaces/Muon Analysis 2.rst

.. _MuonAnalysis_2-ref:

Frequency Domain Analysis
=========================

.. image::  ../images/MuonAnalysis_2.png
   :align: right
   :height: 400px

.. contents:: Table of Contents
  :local:

Interface Overview
------------------

> This interface is used to analyse the data collected on Muon instruments. The interface can be

Reviewer: analyse the time domain data collected ...

> This interface is used to analyse the data collected on Muon instruments. The interface can be

Author: ```suggestion
This interface is used to analyse the time data collected on Muon instruments. The interface can be
```

Reviewer: ```suggestion
This interface is used to analyse the time domain data collected on Muon instruments. The interface can be
```

accessed from the main menu of MantidPlot, in *Interfaces → Muon → Muon Analysis 2*. For sample
datasets, please visit `Muon - Downloads <http://www.isis.stfc.ac.uk/groups/muons/downloads/downloads4612.html>`_.

Loading
-------

The loading section is used to load data as well as to specify what the current runs of interest are.

> **Load Current Run** This will load the latest run for the selected instrument. You must be connected to the

Author: ```suggestion
**Load Current Run** This will load the latest run for the selected instrument if Mantid is connected to the
```

> ISIS data archive for this to work. It also currently only works on windows.

Reviewer: This will load the latest run for the selected instrument if Mantid is connected to the archive. It currently only works on windows.

> ISIS data archive for this to work. It also currently only works on windows.

Author: ```suggestion
archive. It currently only works on windows.
```

> ISIS data archive for this to work. It also currently only works on windows.

Reviewer: ```suggestion
 data archive. It also currently only works on windows.
```

> **Run selection box** This takes a comma seperated list of run numbers and loads them all in. You may also

Author: ```suggestion
**Run selection box** This takes a comma seperated list of run numbers and loads them all in. A range of runs may also
```

> **Run selection box** This takes a comma seperated list of run numbers and loads them all in. You may also

Reviewer: ```suggestion
**Run selection box** This takes a comma seperated list of run numbers for the GUI to load. A range 
```

> specify ranged of runs with a dash. For example ``62260, 62270-3`` will load ``62260, 62270, 62271, 62273``. This also specifies

Reviewer: a range

> specify ranged of runs with a dash. For example ``62260, 62270-3`` will load ``62260, 62270, 62271, 62273``. This also specifies

Author: ```suggestion
be specified with a dash. For example ``62260, 62270-3`` will load ``62260, 62270, 62271, 62273``. This specifies
```

> specify ranged of runs with a dash. For example ``62260, 62270-3`` will load ``62260, 62270, 62271, 62273``. This also specifies

Reviewer: ```suggestion
of runs can be provided by using a dash. For example ``62260, 62270-3`` will load ``62260, 62270, 62271, 62273``. This also specifies
```

> which runs are used by the rest of the GUI.

Reviewer: This specifies the runs that are used in the rest of the GUI.

> which runs are used by the rest of the GUI.

Author: ```suggestion
the runs that are used in the rest of the GUI.
```

> **Browse** This lets you browse locally for files to load.

Author: ```suggestion
**Browse** This opens a dialog window that allows local files to be loaded.
```

**Clear All** This clears all the data from the current GUI.

> **Co-Add** If selected the list of runs chosen are added rather than being loaded individually.

Reviewer: ```suggestion
**Co-Add** If selected the runs are added together for improved statistics. 
```
